contrib/utilities/check_indentation.sh

```diff
@@ -35,5 +35,7 @@ else
 fi
 
 ./contrib/utilities/indent-all
+retval=$?
```

Comment:
how about you do ``./contrib/utilities/indent-all || exit 1``

Reply:
That was my initial commit but I was concerned that we didn't return the diff. Realizing that we don't indent when the author information is off, this seems to be OK again.